### ISSUE TITLE
A11y_30days_Welcome to 30 Days of Learning_Reflow: In Reflow resolution the "Upcoming" and "Self Paced" menu item present in “Menu hamburger” button is not accessible by Keyboard.

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#178]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yTCS;#A11yMAS;#A11ySev2;#30days;#P3_30days_Dec22;#A11yWCAG2.1;#WCAG1.4.10;#DesktopWeb;#Win11;#ChromiumEdge

### Environment Details:
Application Name: 30days
#URL: https://microsoft.github.io/30daysof/
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)

### Pre-Requisite:
To apply Reflow Resolution, follow the below steps:

Open DevTools in new window by press F12.
Adjust the application browser window size to 320256 pixels using mouse (Change the browser zoom levels accordingly up to 200% to adjust the window pixels to 320*256).
Refresh the page.
### Repro Steps:
1. Hit the URL: https://microsoft.github.io/30daysof/
2. Home page will open then tab to "Menu Hamburger" button present in header section.
3. Observe in Reflow resolution, inside the “Menu Hamburger” button "Upcoming" and "Self Paced" menu item is not accessible by keyboard.
### Actual:
In Reflow resolution inside the “Menu Hamburger” button "Upcoming" and "Self Paced" menu item is not accessible by keyboard. The keyboard focus does not goes to the menu items.
Note – This issue is also observed in 200% resize

### Expected:
In Reflow resolution, inside the “Menu Hamburger” all the controls must be accessible by keyboard. 

### User Impact:
Low vision user will face difficulty while accessing the menu items present in “Menu Hamburger” button.

### Attachment:
![A11y_30days_Welcome to 30 Days of Learning_Reflow In Reflow resolution the Upcoming and Self Paced menu item present in “Menu hamburger” button is not accessible by Keyboard](https://user-images.githubusercontent.com/85863895/209610657-aa5c9c9a-3c84-4f0c-ac7c-1437beedc2aa.gif)

**Comment by nitya** [comment]
Navbar has been simplified to remove submenus - this issue does not apply since related items are no longer visible.

**Comment by nitya** [comment]
@Ashutoshsahoo164 we have decided to unpublish the site and deactivate it from internal listings. Can you verify and close this issue now that website is no longer active?

**Comment by Ashutoshsahoo164** [comment]
As per the above comments , closing this bug.
